packages/lib/src/utils/cohortUtils.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * List of cohorts for which the date should not be displayed.
+ * Any cohort name that appears in this array will be excluded from date display.
+ */
+const excludedCohortsForDateDisplay: string[] = ["à venir", "CLE 2025"];
```

Comment:
```suggestion
const excludedCohortsForDateDisplay: string[] = [COHORTS.AVENIR, "CLE 2025"];
```
meme chose pour l'autre, dans `packages/lib/src/constants/cohort.ts`